help_to_heat/frontdoor/routing/forwards_routing.py

```diff
@@ -246,9 +252,8 @@ def _country_next_page(answers):
     return _unknown_response
 
 
-@_requires_answer(supplier_field)
-def _supplier_next_page(answers):
-    supplier = answers.get(supplier_field)
+# where to send the user post them picking a supplier
+def _post_supplier_pick_next_page(supplier):
```

Comment:
Can this be named better so the comment isn't necessary?

Reply:
looking back I don't think the comment adds much anyway. have removed it & changed the name slightly